pkg/controllers/job/state/running.go

```diff
@@ -56,6 +56,13 @@ func (ps *runningState) Execute(action v1alpha1.Action) error {
 				// when scale down to zero, keep the current job phase
 				return false
 			}
+
+			minSuccess := ps.job.Job.Spec.MinSuccess
```

Comment:
Got the reason why you set it as *int instead of int. I perfer to int and set default value to relicas. That may be better. How do you think about it?

Reply:
I still think this field should be *int. 

`MinSuccess` is an **optional** feature, If the user does not set this field, the logic should same as the original.

Reply:
> I still think this field should be *int.
> 
> `MinSuccess` is an **optional** feature, If the user has not set this field, the logic should same as the original.

Yes, it's reasonable.